namespace java socialite.rpc.standalone
namespace py pysocialite.rpc.standalone

include "lang.thrift"

service StandaloneService {
  oneway void terminate(),
}
